samples/interactive-tutorials/events/import-user-events-big-query.js

```diff
@@ -14,16 +14,20 @@
 
 'use strict';
 
-async function main(datasetId) {
+async function main(datasetId, impersonatedPrincipal) {
```

Comment:
Let's make sure the tests are passing after the refactor.

Reply:
Just to confirm, this change won't require the env variable to be set when we run tests? Normally we don't prefer setting env variables for tests since it can be a difficult resource to set up, but as long as they won't fail without it, or if there's a good way to mock it, I'm ok with it